en/orm/validation.rst

```diff
@@ -386,6 +386,19 @@ You may want to use entity methods as domain rules::
         return $entity->isOkLooking();
     }, 'ruleName');
 
+Creating Custom re-usable Rules
+-------------------------------
+
+You may want to re-use custom domain rules. You can do so by creating your own invokable rule::
+
+    use App\ORM\Rule\IsUniqueWithNulls;
+    // ...
+    public function buildRules(RulesChecker $rules)
+    {
+      $rules->add(new IsUniqueWithNulls(['parent_id', 'instance_id', 'name']));
```

Comment:
You should include the closing `}` so the example is easier for folks to copy and paste.